pkg/constants/constants.go

```diff
@@ -125,4 +125,7 @@ const (
 	BlockchainIDLabel = "BlockchainID: "
 
 	PluginDir = "plugins"
+
+	TimestampFormat = "20060102150405"
+	TmpSnapshotName = "tmp-snapshot"
```

Comment:
is this used?

Reply:
Good catch, I didn't need to change this since I decided to skip restarting the network. I reverted this change.